api/src/badge_logic/badges.js

```diff
@@ -1,51 +1,52 @@
-export default {
+module.exports = {
```

Comment:
Why do we still need this file?

Reply:
You're right, we don't. I'll remove this and the console.log @kamicut